webview/__mocks__/@vscode/webview-ui-toolkit/react.tsx

```diff
@@ -36,3 +36,4 @@ const MockComponentWithChildren: React.FC<PropsWithChildren> = ({
 export const VSCodePanels = MockComponentWithChildren
 export const VSCodePanelTab = MockComponentWithChildren
 export const VSCodePanelView = MockComponentWithChildren
+export const VSCodeTag = MockComponentWithChildren
```

Comment:
[F] Adding tags to the rows fixture meant that this was triggered and broke all of the jest tests.